File: ExtensionBox-main/app/src/main/java/com/extensionbox/app/ExtensionBoxApp.java
package com.extensionbox.app;

import android.app.Application;
import com.google.android.material.color.DynamicColors;

public class ExtensionBoxApp extends Application {
    @Override
    public void onCreate() {
        super.onCreate();
        DynamicColors.applyToActivitiesIfAvailable(this);
    }
}

File: app/src/main/java/com/extensionbox/app/ExtensionBoxApp.kt
package com.extensionbox.app

import android.app.Application
import com.google.android.material.color.DynamicColors

class ExtensionBoxApp : Application() {
    override fun onCreate() {
        super.onCreate()
        try {
            DynamicColors.applyToActivitiesIfAvailable(this)
        } catch (ignored: Exception) {
        }
    }
}
